queries/highlights.scm

```diff
@@ -1,6 +1,24 @@
 (php_tag) @tag
 "?>" @tag
 
+
+; Variables
+
+(relative_scope) @variable.builtin
+
+(variable_name) @variable
+
+((name) @constant
+ (#match? @constant "^_?[A-Z][A-Z\\d_]+$"))
+((name) @constant.builtin
+ (#match? @constant.builtin "^__[A-Z][A-Z\d_]+__$"))
+
+((name) @constructor
+ (#match? @constructor "^[A-Z]"))
+
+((name) @variable.builtin
+ (#eq? @variable.builtin "this"))
```

Comment:
probs want this lower, around where it was before maybe

Reply:
Ah, I was trying to keep the section together but I can move it---none of the tests broke though (but they're not the most complete)

Reply:
yeah, i was just going off my gut instinct tbh, since this probably wants to override almost every case